app/models/lesson.rb

```diff
@@ -3,5 +3,5 @@ class Lesson < ActiveRecord::Base
 	mount_uploader :video, VideoUploader
 
 	include RankedModel
-	ranks :row_order, :with_same => :course_id
+	ranks :row_order, :with_same => :section_id
```

Comment:
Lessons do not have a `course_id`.  Instead they have `section_id`'s and that how it should be grouped by via the `RankedModel` gem.